parm/config/config.atmanal

```diff
@@ -0,0 +1,24 @@
+#!/bin/ksh -x
```

Comment:
might as well use `bash` here and all `config` files.

Reply:
New config files have been converted to `bash`